lnrpc/lightning.proto

```diff
@@ -746,6 +746,10 @@ message GetTransactionsRequest {
 
     // An optional filter to only include transactions relevant to an account.
     string account = 3;
+
+    // A list of transaction hashes. When using REST, this field must be
```

Comment:
"What does this list of transaction hashes do?" Would a reader of the API docs ask... Maybe call it `filter_tx_hashes` and describe it that way?